backend/routes/projectById.js

```diff
@@ -0,0 +1,34 @@
+const { parse } = require("url");
+
+const { missingProperty, invalidProperty } = require('../src/utils/errors');
+const stringify = require('../src/utils/stringify');
+const getAllProjects = require('../src/components/projects/getAllProjects');
+
+module.exports = async (req, res) => {
+  const { query: { project: projectParam } } = parse(req.url, true);
+
+  if (!projectParam) {
+    res.statusCode = 422;
+    res.end(stringify({
```

Comment:
Actually I am not sure why but express is recommending to use send method to end the request with some payload. https://expressjs.com/en/api.html#res.end
Also I am pretty confident that Content-Type header for json payload will not be sent here. You should actually use `.json()` method because it stringifies object for you and adds this header.

Reply:
Let's remember that those are not express endpoints, but rather serverless lambdas for which you receive request from standard `http` package, no from `express`. But still you've got the point that I didn't think about `Content-Type`.

Reply:
Totally forgot, thanks